addons/SpellBook/SpellBook.lua

```diff
@@ -0,0 +1,382 @@
+_addon.name     = 'SpellBook'
+_addon.author   = 'SigilBaram'
+_addon.version  = '0.0.1'
+_addon.commands  = {'spellbook','spbk'}
+
+require('tables')
+res = require('resources')
+
+spell_types = {
+    whitemagic= { type = 'WhiteMagic', readable = 'White Magic spells' },
+    blackmagic= { type = 'BlackMagic', readable = 'Black Magic spells' },
+    songs=      { type = 'BardSong', readable = 'Bard songs' },
+    ninjutsu=   { type = 'Ninjutsu', readable = 'Ninjutsu' },
+    summoning=  { type = 'SummonerPact', readable = 'Summoning spells' },
+    bluemagic=  { type = 'BlueMagic',  readable = 'Blue Magic spells' },
+    geomancy=   { type = 'Geomancy', readable = 'Geomancy spells' },
+    trusts=     { type = 'Trust', readable = 'Trusts'},
+    all=        { type = 'all', readable = 'spells of all types'}
+}
```

Comment:
Drives me nuts that you aligned the start of the table but not the individual entries and that you didn't insert a space before the first `=` :D